ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/ImagesTable.tsx

```diff
@@ -146,19 +153,24 @@ function ImagesTable({
                             <Tr>
                                 <Td dataLabel="Image">
                                     {name ? (
-                                        <ImageNameTd name={name} id={id}>
-                                            {isWatchedImage && (
-                                                <Label
-                                                    isCompact
-                                                    variant="outline"
-                                                    color="grey"
-                                                    className="pf-u-mt-xs"
-                                                    icon={<EyeIcon />}
-                                                >
-                                                    Watched image
-                                                </Label>
-                                            )}
-                                        </ImageNameTd>
+                                        <ImageScanningErrorLabelLayout
+                                            imageNotes={notes}
+                                            scanNotes={scanNotes}
+                                        >
```

Comment:
I'm interested to know why you chose this pattern to inject the optional scan notes tag this way, by putting it in a "slot" at the bottom of this wrapper element, with the original custom table-data component as the child that goes in the top "slot". It's an unexpected source order.

Reply:
Good question. Initially, I wanted to go with this approach because of composition and separating concerns. Wrapping `ImageNamesTd` with this new component (showing a label at the bottom) adheres to the principle of composition. It also separates concerns because it has one function: displaying the image scanning error label at the bottom. If we ever decided to reuse this component in other places, the name of the component would make it easier to reuse. 

Coming back to this again, I do have a few more suggestions for improvement and/or different approaches we could do here. I can list them out:
1. We can improve the naming of some of these functions.  Technically, `ImageNameTd` doesn't mention anything about being a table data cell so we could rename that to be more generic.
2. `ImageNameTd` currently acts like a layout manager. We could technically refactor that component and this new component to make one component that deals with the layout. All the pieces in the layout could become their own components (ie. Copy button, Image name link, etc.). That would also make those components more reusable.
3. I realized that the `ImageNameTd` component technically could just take the label as a child component. Must have overlooked that. I could just rewrite this new component and plug it in there. Option 1 and 2 could still be a consideration though

Reply:
Updating the PR with some improvements. Going with Option 3.